xrpackage-api.js

```diff
@@ -0,0 +1,557 @@
+import * as THREE from './xrpackage/three.module.js';
```

Comment:
I might call this whole file `xrpackage-package.js`; the API includes the engine, not just this file, which largely implements `XRPackage`.